test/Microsoft.ML.TestFramework/TestCommandBase.cs

```diff
@@ -2154,8 +2154,8 @@ public void SavePipeChooseColumnsByIndex()
         [Fact()]
         public void SavePipeTextLoaderWithMultilines()
         {
-            string dataPath = GetDataPath("multiline.csv");
-            const string loaderArgs = "loader=text{sep=, quote+ multilines+ header+ col=id:Num:0 col=description:TX:1 col=animal:TX:2}";
+            string dataPath = GetDataPath("multiline-escapechar.csv");
+            const string loaderArgs = "loader=text{sep=, quote+ multilines+ header+ escapechar=\\ col=id:Num:0 col=description:TX:1 col=animal:TX:2}";
```

Comment:
For my tests I've reused the multiline.csv file I introduced on PR #5125, and simply created a multiline-escapechar.csv where escaped quotes are marked with \". Since both files are the same, except for the escapeChar, they're loaded into the same resulting DataView, and the output baselines for both datasets are the same (so there was no need to add more baselines).